spec/shared_examples/shared_tests.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+
+shared_examples_for 'shared_tests' do
+
+
+  describe "GET 'index'" do
+    it "is successful" do
+      get :index
+      expect(response.status).to eq 200
+    end
+  end
+
+  describe "GET 'new'" do
+    it "renders the new page" do
+      get :new
+      expect(subject).to render_template :new
+    end
+  end
+
+  describe "GET 'edit'" do
+    it "renders edit page" do
+      new_medium = model.create(title: "Hello, World!", creator: "Jennie")
+      get :edit, id: new_medium.id
+      expect(subject).to render_template :edit
+    end
+  end
+
+  describe "GET 'show'" do
+    it "renders show page" do
+      new_medium = model.create(title: "Hello, World!", creator: "Jennie")
+      get :show, id: new_medium.id
+      expect(subject).to render_template :show, id: new_medium.id
+    end
+  end
+
+  describe "POST 'create'" do
+    it "redirects to show on success" do
+      post :create, good_params
+      new_medium = model.last
+      expect(subject).to redirect_to polymorphic_path(new_medium) # need to fix this path
```

Comment:
Not sure if you know – but if you write your comments like so: `#TODO: fix this path` you can run `rake notes` in your project directory in Terminal and it'll show you all the lines that include `FIXME` and `TODO` so you don't miss comments like these. See [the documentation](http://guides.rubyonrails.org/command_line.html#notes).